tests/network/vmi_multus.go

```diff
@@ -332,26 +353,43 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, cidr1, cidr2 string) {
```

Comment:
I wouldn't use `cidr1` and `cidr2` as the variable names here - I would use `staticIPVm1` and `staticIPVm2`. You could pass an empty string when you want to have dhcp (or even the dhcp string you're using now).

Reply:
Done.